src/OpenLoco/src/Ui/WindowManager.h

```diff
@@ -189,7 +182,7 @@ namespace OpenLoco::Ui::Windows
 
     namespace Error
     {
-        void open(StringId title, StringId message);
+        void open(StringId title, StringId message = StringIds::null, bool playSound = true);
```

Comment:
Rather than have a bool playSound = true i'd prefer if we just had a `openQuiet` function. I hate seeing functions where the bool argument is not clear.

Reply:
Sure, sounds sensible. I'll rework it.